lib/services/nodes-api-service.js

```diff
@@ -509,6 +509,26 @@ function nodeApiServiceFactory(
     };
 
     /**
+     * Remove the tag from a list of nodes
+     * @param  {String}     tagName
+     * @return {Promise}
+     */
+    NodeApiService.prototype.masterDelTagById = function(tagName) {
+        return Promise.resolve().then(function() {
+                assert.string(tagName, 'tag must be a string');
```

Comment:
@zyoung51: This function is called in a controller callback which is always called inside a promise chain so this error would be caught by the .catch in the controller wrapper function.  That said, I think you're right.  We should probably just leave this function as is since in case it is ever used outside a controller.